certbot/client.py

```diff
@@ -279,16 +279,36 @@ def obtain_certificate(self, domains):
         :rtype: tuple
 
         """
+
+        # We need to determine the key path, key PEM data, CSR path,
+        # and CSR PEM data.  For a dry run, the paths are None because
+        # they aren't permanently saved to disk.  For a lineage with
+        # --reuse-key, the key path and PEM data are derived from an
+        # existing file.
+
+        if oldkeypath is not None:
+            # We've been asked to reuse a specific existing private key.
+            # Therefore, we'll read it now and not generate a new one in
+            # either case below.
+            with open(oldkeypath, "r") as f:
+                keypath = oldkeypath
+                keypem = f.read()
+            key = util.Key(file=keypath, pem=keypem)
+            logger.info("New key obtained by reusing existing private key "
```

Comment:
Perhaps clearer to drop the "new key obtained by"

Reply:
I didn't think so because it does create a new file, but maybe users will be able to figure it out!